examples/lock-app/linux/README.md

```diff
@@ -15,3 +9,57 @@ The application could be build in the same manner as `all-clusters-app`:
 ```
 ? scripts/examples/gn_build_example.sh examples/lock-app/linux out/lock-app chip_config_network_layer_ble=false
 ```
+
+# Named Pipe Interface
+
+This application accepts commands over named pipe. Pipe is located in
+`/tmp/chip_lock_app_fifo-<PID>`.
+
+## Command Format
+
+Commands interface accepts commands formatted as a JSON object. Structure of the
+object:
+
+| Key      | Type   | Value Description                                                                               | Optional |
+| -------- | ------ | ----------------------------------------------------------------------------------------------- | -------- |
+| `Cmd`    | String | Contains name of the command to execute                                                         | No       |
+| `Params` | Object | Contains parameters for the command. Could be omitted if the command does not accept parameters | Yes      |
+
+For example:
+
+-   `{ "Cmd": "SetDoorState", "Params": { "EndpointId": 1, "DoorState": 2 } }` -
+    command to set the Door Jammed door state.
+-   `{ "Cmd": "SendDoorLockAlarm", "Params": { "AlarmCode": 0 } }` - command to
+    send the Door Lock Alarm event with Lock Jammed alarm code.
+
+## Available commands
+
+### Set Door State
+
+-   Name: `SetDoorState`
+-   Parameters:
+    -   `EndpointId` (Uint, Optional): ID of the endpoint where command should
+        be executed. Could be omitted, in that case the default endpoint with ID
+        1 is targeted.
+    -   `DoorState` (DoorState enum): new door state.
+-   Usage:
+    ```bash
+     echo '{"Cmd": "SetDoorState", "Params": { "EndpointId": 1, "DoorState": 1 } }' > /tmp/chip_lock_app_fifo-<PID>
```

Comment:
nit: Replace "Cmd" with "Name" to align with all-clusters-app

Reply:
My initial thought was that this is different app and it doesn't have to be compliant with the all-clusters-app given that documentation is in place. If you strongly prefer the 'Name' instead of 'Cmd', I can make the change.

Reply:
Given where we are in the schedule, marking this as resolved. @Morozov-5F Please feel to follow up if you feel strongly.